### ISSUE TITLE
Chrome: storage get for 'should_hash' always returns false

### ISSUE THREAD

**Issue by crashdemons** [crashdemons/3301-hash-alarm#7]
even after hitting the enable button, `should_hash` remains false.

**Comment by crashdemons** [comment]
logs show that storage initialization happens from Request code like expected:
```
Request:setStorageItem 
   Object
   config: {mode: "most", enabled: false}
```

`chrome.storage.local.get('config',function(v){ console.log("v",v); })`
still returns empty though:
`v {}`

so even though set occurs, it doesn't seem to properly initialize the storage

**Comment by crashdemons** [comment]
![image](https://user-images.githubusercontent.com/3233238/113791870-39325880-970a-11eb-94d1-0b7aa26fe702.png)

**Comment by crashdemons** [comment]
fixed by c499575ee47b2ca1d674520f38091a3be5621716
